libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/MultiLanguageRecognizer.cs

```diff
@@ -55,7 +55,7 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dialog
                 policy.AddRange(targetpolicy);
             }
 
-            if (activity.Locale.Length != 0 && LanguagePolicy.TryGetValue(string.Empty, out string[] defaultPolicy))
+            if (activity.Locale?.Length != 0 && LanguagePolicy.TryGetValue(string.Empty, out string[] defaultPolicy))
```

Comment:
>activity.Locale?.Length != 0 && [](start = 16, length = 31)

there should be no check against activity.Locale at all, remove this.

Reply:
resolved.